compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.td

```diff
@@ -23,8 +23,20 @@ def AMDAIEBufferizeToAllocation :
   let options = [
     Option<"memorySpace", "memory-space", "int64_t", /*default=*/"1",
       "Set the memory space">,
-    Option<"bufferizeLevel", "bufferize-level", "int64_t", /*default=*/"-1",
-      "A temporary workaround to figure the config for bufferization">
+    Option<"bufferizeOperand", "bufferize-operand",
+      "mlir::iree_compiler::AMDAIE::BufferizeOperand",
+      /*default=*/"mlir::iree_compiler::AMDAIE::BufferizeOperand::InputOutput",
+      "Select which operands of a matmul to be bufferized to allocation",
```

Comment:
`matmul` -> `linalg op` - since the pass can be applied to literally any ContractionOpInterface LinalgOp.

Reply:
I see this wasn't addressed - marking it unresolved.